protocol/messenger_walletconnect.go

```diff
@@ -0,0 +1,47 @@
+package protocol
+
+import (
+	"context"
+
+	"github.com/status-im/status-go/protocol/requests"
+)
+
+type WalletConnectSession struct {
+	PeerID   string `json:"peerId"`
+	DAppName string `json:"dappName"`
+	DAppURL  string `json:"dappURL"`
+	Info     string `json:"info"`
+}
+
+func (m *Messenger) getWalletConnectSession() ([]WalletConnectSession, error) {
+
+	response, err := m.persistence.GetWalletConnectSession()
+	if err != nil {
+		return response, err
+	}
+	return response, err
+}
+
+func (m *Messenger) AddWalletConnectSession(ctx context.Context, request *requests.AddWalletConnectSession) error {
```

Comment:
The `ctx` parameter can be removed from `AddWalletConnectSession`, `GetWalletConnectSession` and `DestroyWalletConnectSession`, since it's not being used